dockstore-client/src/main/java/io/dockstore/client/cli/nested/ToolClient.java

```diff
@@ -294,7 +294,7 @@ protected void handleListNonpublishedEntries() {
     @Override
     protected void handleListUnstarredEntries() {
         try {
-            List<DockstoreTool> containers = containersApi.allPublishedContainers();
+            List<DockstoreTool> containers = containersApi.allPublishedContainers(null, null, null, null, null);
```

Comment:
Would default values make more sense?

Reply:
The problem is that containersApi is a generated class (generated from our webservice's yaml). 
Unless there's a way to configure the codegen (and I don't want to deviate too much from their templates) we don't have control over their api (which really sucks in this case since all the values are optional).